_posts/2021-03-04-geoserver-2-19-rc-released.md

```diff
@@ -0,0 +1,195 @@
+all requests, past and ongoing, can be stored in a database, for later reference.
+
+The module uses GeoTools JDBC stores to access databases, create the necessary tables, and track status. 
+Connection parameters are provided as property files, e.g.:
+
+
+```
+user=postgres
+port=5432
+password=******
+passwd=******
+host=localhost
+database=gsstore
+driver=org.postgresql.Driver
+dbtype=postgis
+```
+
+For more information, refer to the [module documentation](https://docs.geoserver.org/latest/en/user/extensions/wps-jdbc/index.html).
+
+We'd like to thank Ian Turton for developing the module on behalf of GeoSolutions, Alessio Fabiani for providing
+documentation for it, and Andrea Aime for performing the QA and graduation steps.
+
+## WPS Download extension
+
+The WPS download plugin provides processes assisting in the download of large amounts of data,
+allowing to use asynchrounous request, when usage of WFS, WCS or WMS for the same job would lead to HTTP timeouts.
+Also, download limits can be configured to avoid excessively large requests: size in MB, number of features, number
+of animation frames.
+
+In particular, the following processes are available:
+
+*  ``DownloadEstimator``, verifying that a raster/vector download about to be attempted will fit the download limits.
+* ``DownloadProcess``, allowing to download either raster or vector data, reproject and clip them
+* ``DownloadMapProcess``, allows to download a large map matching what is visible on a client (which may be using tiles and display on a multi-screen), eventually dynamically fetching layers from remote WMS servers as well. It's also possible to decorate the final map using the standard [decoration layouts](https://docs.geoserver.org/latest/en/user/services/wms/decoration.html#wms-decorations).
+* ``DownloadAnimationProcess``, allows to build a MP4 movie given a set of layers and times.
+
+(screen snap)
+
+For more information, refer to the [module documentation](https://docs.geoserver.org/latest/en/user/extensions/wps-download/index.html).
+
+Thanks to Alessio, Andrea, Daniele, from GeoSolutions, for developing the extension, and GeoNode/MapStore for testing it in various production environments.
+
+## WMTS Multidimensional extension
+
+The WMTS multi-dimensional extension is an extension to the WMTS protocol developed during OGC Testbed 12. The extension allows to explore 
+the dimensions attached to a dataset, prodiving ways to explore them, finding relationships between them.
+
+Here are a couple of real world examples of this functionality:
+
+* GeoServer is publishing a set of satellite images. Eeach image is time stamped. The user is browsing the set of data on a map, the client wants to show the list of available times for the current area. The WMS/WMTS dimension support cannot help, but the WMTS extension has a request, ``GetDomainValues``, which allows to answer exactly this question.
```

Comment:
* GeoServer is publishing a set of satellite images. Each image is time stamped. The user is browsing the set of data on a map, and the client software wants to show the list of available times for the current area. The WMS/WMTS dimension support cannot help, but the WMTS extension has a request, ``GetDomainValues``, which exactly answers this question.